app/controllers/setup_controller.rb

```diff
@@ -58,6 +58,9 @@ def do_bootstrap
       redirect_to setup_bootstrap_path, alert: res
       return
     end
+    Pillar.find_or_create_by pillar: "api:etcd_version" do |pillar|
+      pillar.value = "etcd3"
+    end
```

Comment:
This is created for consistency reasons, so all installations have the `api:etcd_version` override. Old installations will have `etcd2` because of the migration, while new installations will have `etcd3` because of this.

I still have my doubts about the need of a consistent overlay between versions, but included so it's possible to discuss this.